Tests/FluentAssertions.Specs/Equivalency/CollectionEquivalencySpecs.cs

```diff
@@ -2830,5 +2830,77 @@ public static IEnumerable<object[]> ArrayTestData()
                    from y in arrays
                    select new object[] { x, y };
         }
+
+        [Fact]
+        public void Comparing_lots_of_complex_objects_should_still_be_fast()
```

Comment:
Failed in pipeline

Reply:
Yeah, AppVeyor is a lot slower. Had to increase the threshold to 20 seconds.